kubernetes.yaml

```diff
@@ -106,6 +120,34 @@ metadata:
     app.kubernetes.io/name: spider-example
 stringData:
   FOO_API_KEY: "1234567890abcdef"
+#---
+#apiVersion: v1
+#kind: PersistentVolume
+#metadata:
+#  name: efs-pv
+#spec:
+#  capacity:
+#    storage: 5Gi
+#  volumeMode: Filesystem
+#  accessModes:
+#    - ReadWriteOnce
+#  storageClassName: ""
+#  persistentVolumeReclaimPolicy: Retain
+#  csi:
+#    driver: efs.csi.aws.com
+#    volumeHandle: fs-04a902f365d2d62bd
+#---
+#apiVersion: v1
+#kind: PersistentVolumeClaim
+#metadata:
+#  name: efs-claim
+#spec:
+#  accessModes:
+#    - ReadWriteOnce
+#  storageClassName: ""
+#  resources:
+#    requests:
+#      storage: 5Gi
```

Comment:
Good you added these (commented out) to the manifest. Please reduce is to the minimum needed to run on minikube (which is used on CI). Even though this is commented out, I think it's good to tune to that use-case, then people can adapt to their own cloud provider (so no mention of efs).